### DISCUSSION TITLE
TS-XEL

### DISCUSSION THREAD

**Discussion by ErrorUserNameNotFound404** [1000001101000/Debian_on_Buffalo#183] [General]
Hello,

So I have the TS-XEL and ive attempted to use the files in bookworm and bullseye

for reference im installing:
initrd.buffalo
uImage-88f6281.buffalo.tsxel > uImage-88f6281.buffalo

I am able to insert the files with the js script, but am getting lost init at reboot
I then attempt with the telnet way and am getting the same results of lost init  at reboot.

Any help with this?

**Comment by 1000001101000** [comment]
What size are the boot files that you loaded?

**Reply by ErrorUserNameNotFound404** [reply]
Thanks for fast response!
initrd.bufallo is 145k
uImage-88f6281.buffalo is 146k

Now that im looking at the file on here it seems theyre undersized as they should be a few mb. I used wget, let me tinker again.

**Reply by 1000001101000** [reply]
Usually means you downloaded the page for the file rather than the file itself, it should work if you click through then grab the “raw” like for your wget command.

**Reply by ErrorUserNameNotFound404** [reply]
Makes sense, im still new to git/github :)

**Reply by 1000001101000** [reply]
Any luck?

**Reply by ErrorUserNameNotFound404** [reply]
Yep, that was the issue. 

Almost all done now. Just setting up samba users are getting the original xfs into a share.

Thanks for the help and then documentation.
